scripts/qa_crowbarsetup.sh

```diff
@@ -639,17 +639,28 @@ function addcloud6pool()
 
 function addcctdepsrepo()
 {
+    if [[ "$cloudsource" == @(develcloud5|GM5|GM5+up) ]]; then
+        zypper ar -f http://$susedownload/ibs/Devel:/Cloud:/Shared:/Rubygem/SLE_11_SP3/Devel:Cloud:Shared:Rubygem.repo
+    else
+        add_sdk_repo
+    fi
+}
+
+function add_sdk_repo()
```

Comment:
I like this "extract function" refactoring. Personally I would have put it in a separate commit for the reasons stated in https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes.